synthesis/population/spatial/primary/candidates.py

```diff
@@ -129,14 +141,22 @@ def execute(context):
     )
 
     df_locations = context.stage("synthesis.locations.education")
-    df_education = process(context, "education", random, df_persons,
-        df_education_od, df_locations
-    )
+    if context.config("education_location_source") == 'bpe':
+        df_education = process(context, "education", random, df_persons, df_education_od, df_locations)
+    else :
+        df_education = []
+        for prefix, education_type in EDUCATION_MAPPING.items():
+            df_education.append(
+                process(context, "education_" + prefix, random,
```

Comment:
I see why you do the `split` above, but why not just pass "education" here? I would rather add another argument for the logging purposes, like `step_name` and then use it in the prints in `process`.